apps/domain/src/main/core/infrastructure/providers/provider.py

```diff
@@ -17,7 +17,7 @@
 
 class Provider:
     def __init__(self, config):
-        folder_name = f"{config.provider}-{config.app.name}-{config.app.id}"
```

Comment:
We need to have `config.app.id` in the folder name, so as to separate workers terraform config files into different folders.

Reply:
Done.